sky/sdk/lib/widgets/block_viewport.dart

```diff
@@ -155,8 +198,43 @@ class BlockViewport extends RenderObjectWrapper {
     }
   }
 
+  // Build the widget at index, and use its maxIntrinsicHeight to fix up
+  // the offsets from index+1 to endIndex. Return the newWidget.
+  Widget _recomputeOffsets(int index, int endIndex, BoxConstraints innerConstraints) {
```

Comment:
I guess my point is "recomputeOffsets" isn't the name of a method that returns a Widget. It sounds like a method that either returns void or returns offsets.